File: C/circulo.c
#include <stdio.h>
#include <math.h>

int main() {

    double x, area;

    printf("Digite o valor do raio do circulo: ");
    scanf("%lf", &x);

    area = pow(x, 2) * M_PI;

    printf("AREA = %.3lf", area);

    return 0;
}

File: Java/circulo.java
import java.util.Locale;
import java.util.Scanner;

public class Circulo {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        double raio, area;

        System.out.print("Digite o valor do raio do circulo: ");
        raio = sc.nextDouble();

        area = Math.PI * Math.pow(raio, 2);

        System.out.println("AREA = " + String.format("%.3f", area));

        sc.close();
    }
}